ThePensionsRegulator.Frontend/Styles/_tpr-file-upload.scss

```diff
@@ -0,0 +1,36 @@
+﻿.govuk-file-upload {
+    font-size: 1rem;
+    outline: 3px solid transparent;
+    border-color: transparent;
+    box-shadow: inset 0 0 0 2px transparent;
+    padding: 3px 3px 1px 3px;
+    font-family: $govuk-font-family;
+    color: $govuk-text-colour
+}
+
+.govuk-file-upload::file-selector-button {
+    padding-top: 8px;
+    padding-bottom: 7px;
+    padding-left: $tpr-button-padding;
```

Comment:
I would expect top and bottom padding to be the same as `.govuk-button` and therefore set in the same way. Top is `(govuk-spacing(2) - $govuk-border-width-form-element)` and bottom is `(govuk-spacing(2) - $govuk-border-width-form-element - ($button-shadow-size / 2))`. See https://github.com/alphagov/govuk-frontend/blob/main/packages/govuk-frontend/src/govuk/components/button/_index.scss

When govuk-frontend changes we want to adapt automatically and consistently. Hard-coded values prevent that.

Reply:
done